src/FSharpVSPowerTools.Logic/GoToDefinitionFilter.fs

```diff
@@ -421,10 +432,21 @@ type GoToDefinitionFilter(textDocument: ITextDocument,
                 match symbolResult with
                 | Some (_, _, _, _, FSharpFindDeclResult.DeclFound _) 
                 | None ->
-                    // Run the operation on UI thread since continueCommandChain may access UI components.
-                    do! Async.SwitchToContext uiContext
-                    // Declaration location might exist so let Visual F# Tools handle it. 
-                    return continueCommandChain()
+                    // no FSharpSymbol found, here we look at #load directive
```

Comment:
Got it thanks!

I'm trying to see if I should remove `Async.RunSynchronously` I've put in `GoToDefinitionTests.LoadDirective.fs` `getAst`, is it something we are concerned with?